packages/core/src/components/menu/menuDivider.tsx

```diff
@@ -11,23 +11,26 @@ import * as Classes from "../../common/classes";
 import { IProps } from "../../common/props";
 
 export interface IMenuDividerProps extends IProps {
-    /** Optional header title. */
-    title?: string;
+    /** Optional header title. Can also be passed as `children`. */
```

Comment:
why both? it seems unnecessary

Reply:
it does indeed. but `children` was supported and unused. how would you like to see this resolved?

Reply:
it was supported but not documented? then just remove it and say it's a breaking change

Reply:
the `children` prop was allowed, but was not rendered. now it is rendered.

Reply:
cool, I'd rather remove the broken feature than add proper support for it

Reply:
wait... so... `children?: never` or remove `title` prop?

Reply:
tbh I don't think typescript is able to fail on children passed to components with `children: never` props in `.tsx` files, so it's purely for documentation